app/request_handler.py

```diff
@@ -61,27 +77,6 @@ def writeSearchRecord(lat, lng, key=None):
     record["time"] = time.time()
     db.child(searchesTable).update({ record["g"]: record })
 
-def findSearchRecord(center, radius=1000):
-    import app.geofire as geo
-    import time
-    queries = geo.geohashQueries(center, radius)
-    now = time.time()
-
-    for query in queries:
-        results = db.child(searchesTable).order_by_key().start_at(query[0]).end_at(query[1]).get()
-        for result in results.each():
-            record = result.val()
-            if record.get("time", 0) + searchCacheExpiry < now:
-                db.child(searchesTable).child(result.key()).remove()
-                continue
-            # double check that we're within distance
-            circleDistance = geo.distance(center, record["l"]) * 1000
-            # 1000 m in 1 km (geo.distance is in km, searchCacheRadius is in m)
-            if circleDistance < searchCacheRadius:
-                return record
-            log.info("Circle distance is " + str(circleDistance))
-
-
 def readCachedVenueDetails(key):
```

Comment:
Yeah, I left this work as a follow-up. `researchVenue` still calls `readCachedVenueDetails`, but we no longer call `researchVenue` at all with these changes. I figured we'd fix/clean up these methods as we reintroduce them in `crawl_expand.py`.